### PR TITLE
Upgrade inspectpack

### PR DESCRIPTION
**Pull request by tptee**
This parallelizes some of the inspectpack actions and improves the cache!

### CODE REVIEW

File: plugin/index.js [hunk 1]
```diff
@@ -38,16 +38,16 @@ function observeBundleMetrics(stats, inspectpack) {
   const bundlesToObserve = Object.keys(stats.compilation.assets)
     .filter(
       bundlePath =>
-        // Don't include hot reload assets, they break everything
+        // Don"t include hot reload assets, they break everything
```
Comment (ryansully) [inline plugin/index.js]: Your linter/editor seems to be unnecessarily changing all the single quotes to double quotes, as well as removing dangling commas.

**Comment by kenwheeler** [comment]
👍

**Comment by ryan-roemer** [comment]
@tptee -- Can you add a `.travis.yml` with Node `4`, `6`, `8` to this PR so that we get the lint thing solved once and for all?

@ryansully -- Good eye spotting the over aggressive search and replaces on quote chars!

File: plugin/index.js (hunk 1 above)
Comment (tptee) [reply plugin/index.js]: Ah, I find-and-replaced too hard here. The reverse is true though: a previous commit must've formatted this with custom Prettier plugin settings or something. This corrects it to fit into our lint rules.

**Comment by tptee** [comment]
@ryan-roemer done! Can you enable the repo in Travis? I don't have permission

**Comment by ryan-roemer** [comment]
@tptee -- Enabled. I'm going to close and re-open this PR to hopefully kick Travis...

**Comment by ryan-roemer** [comment]
Looks great and CI is now passing!

**Pull request merged**